ci/build/build-docker-images.py

```diff
@@ -62,7 +62,7 @@
 # If changing the CUDA version in the below line, you should also change the base Docker
 # image being used in ~/ci/docker/Dockerfile.gpu to match the same image being used
 # here.
-ML_CUDA_VERSION = "cu116"
+ML_CUDA_VERSION = "cu117"
```

Comment:
Please adhere to the comment and also change `/ci/docker/Dockerfile.base.gpu` (the comment is a bit off, feel free to correct it)